firmware/common/lcd_ili9341.hpp

```diff
@@ -100,6 +100,9 @@ class ILI9341 {
     constexpr ui::Dim height() const { return 320; }
     constexpr ui::Rect screen_rect() const { return {0, 0, width(), height()}; }
 
+    void draw_pixels(const ui::Rect r, const ui::Color* const colors, const size_t count);
```

Comment:
Why did you need to move these? Aren't the templated ones the same but with size deduction?

Reply:
Was it because you didn't have a statically known array size?

Reply:
Yes that is why

Reply:
I tried with the array suitable for the largest font size 8x16, but the templated ones always read/write the size of the array, whereas I need to adjust the read/write pixel count based on the active font.  I could allocate a separate array for each font size, but that seemed unnecessary when I could just call this existing function.